examples/plot_bcic_iv_4_ecog_cropped.py

```diff
@@ -0,0 +1,358 @@
+"""
+Fingers flexion cropped decoding on BCIC IV 4 ECoG Dataset
+==========================================================
+
+Building on the Trialwise decoding tutorial, we now do more data-efficient cropped decoding!
+
+In Braindecode, there are two supported configurations created for training models: trialwise
+decoding and cropped decoding. We will explain this visually by comparing trialwise to cropped
+decoding.
+"""
+
+
+# Authors: Maciej Sliwowski <maciek.sliwowski@gmail.com>
+#          Mohammed Fattouh <mo.fattouh@gmail.com>
+#
+# License: BSD (3-clause)
+
+
+######################################################################
+# Loading and preprocessing the dataset
+# -------------------------------------
+#
+
+
+######################################################################
+# Loading
+# ~~~~~~~
+#
+
+
+######################################################################
+# First, we load the data. In this tutorial, we use the functionality of
+# braindecode to load datasets through
+# `MOABB <https://github.com/NeuroTechX/moabb>`__ to load the BCI
```

Comment:
This text and also a lot of other text in this example is not correct/still fits to old example. Feel free to also shorten text quite a bit, but at least the text that is there should be correct.